src/molecules/formfields/CheckboxWrapper/index.js

```diff
@@ -2,45 +2,153 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
+import ErrorMessage from 'atoms/ErrorMessage';
+import Layout from 'atoms/Layout';
+import Col from 'atoms/Layout/Col';
 import Tooltip from 'atoms/Tooltip';
+import Icon from 'atoms/Icon';
 import styles from './checkbox_wrapper.module.scss';
 
 function renderTooltip(tooltip, className) {
   if (typeof tooltip === 'string') {
```

Comment:
You used this same function in in the `RadioGroup` component. Any way we can extract this into a `util` function?

Reply:
@sunnymis good point, I'll refactor